ambari-server/src/main/resources/Ambari-DDL-Derby-CREATE.sql

```diff
@@ -134,10 +134,9 @@ CREATE TABLE servicegroups (
   service_group_name VARCHAR(255) NOT NULL,
   cluster_id BIGINT NOT NULL,
   stack_id BIGINT NOT NULL,
-  CONSTRAINT PK_servicegroups PRIMARY KEY (id, cluster_id),
-  CONSTRAINT FK_servicegroups_cluster_id FOREIGN KEY (cluster_id) REFERENCES clusters (cluster_id),
-  CONSTRAINT FK_servicegroups_stack_id FOREIGN KEY (stack_id) REFERENCES stack (stack_id),
-  CONSTRAINT UQ_TEMP_UNTIL_REAL_PK UNIQUE(id));
+  CONSTRAINT PK_servicegroups PRIMARY KEY (id),
+  CONSTRAINT UK_servicegroups_id UNIQUE (id, cluster_id, service_group_name),
```

Comment:
This unique constraint is not needed unless you want to remove the ID value so that a service group name is unique in a cluster.

Reply:
We want SG name to be unique across cluster, as the name is the only thing distinguishable from API perspective. CC @jayush